extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -189,6 +189,7 @@ class AmpLightbox extends AMP.BaseElement {
     });
 
     if (this.isScrollable_) {
+      this.element./*OK*/scrollTop = this.element./*OK*/scrollTop || 1;
```

Comment:
Doing this in the `initialize_()` is not working because the lightbox element is not visible yet so the scrollTop adjustment won't take effect.
Instead, in order to adjust `this.element.scrollTop` initially when opening lightbox, we could call `this.scrollHandler_()` around [here](https://github.com/ampproject/amphtml/blob/master/extensions/amp-lightbox/0.1/amp-lightbox.js#L242) after the lightbox becomes visible.